### DISCUSSION TITLE
Screen Height Question

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10124] [Help] [answered]
Waddup ya'll. 

I hope everyone is doing well.

Why isn't my pop-out menu taking up the entire [screen height](https://tailwindcss.com/docs/height)? 

```
<div id="hamburger-menu" class="h-7 w-9 relative group peer z-10">
       <div class="w-full h-1 absolute top-0 bg-yellow-1 transition-all duration-[600ms] ease-linear group-[.clicked]:-rotate-[315deg] group-[.clicked]:top-1/2 group-[.clicked]:-translate-y-1/2"></div>
       <div class="w-full h-1 absolute top-1/2 -translate-y-1/2 bg-red-1 transition-opacity duration-[800ms] ease-linear group-[.clicked]:opacity-0"></div>
       <div class="w-full h-1 absolute top-full -translate-y-1 bg-yellow-1 transition-all duration-[600ms] ease-linear group-[.clicked]:-rotate-[225deg] group-[.clicked]:top-1/2 group-[.clicked]:-translate-y-1/2"></div>
</div>
<div id="pop-out-menu" class="fixed w-3/4 -right-3/4 bg-gray-100 h-screen transition-all duration-500 peer-[.clicked]:right-0">
</div>
```

![Screen Shot 2022-12-19 at 8 39 46 PM](https://user-images.githubusercontent.com/96091818/208561446-c7b60df4-5fb6-4c1f-a3f0-f83f9a8ab0df.png)

**Comment by wongjn** [comment, marked as answer]
The `align-items: center` from the `<nav>` element parent is aligning the `h-screen` element such that it's midpoint is aligned in the center of the main axis of the flex layout. To have it positioned such that its entire height covers the screen, try applying `top: 0` via `top-0` or `bottom: 0` via `bottom-0` to the `h-screen` element.

**Reply by kmarler18** [reply]
@wongjn Thank you Justin!
